Factorial/Javascript/Factorial.js

```diff
@@ -4,4 +4,6 @@ function factorial(num) {
   return num * factorial(num-1);
 }
 
+// Test
+console.log(factorial(0));
```

Comment:
If we're going to add test cases be sure to comment what the expected output should be